test/integration/appDuplicate.spec.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
+import { test, expect } from '../playwright/test';
+import { ToolpadHome } from '../models/ToolpadHome';
+import { readJsonFile } from '../utils/fs';
+import * as locators from '../utils/locators';
+
+test('duplicate app from home flow', async ({ page }) => {
+  const dom = await readJsonFile(path.resolve(__dirname, './duplicateNavigationDom.json'));
```

Comment:
if we're not checking anything about this DOM maybe we don't need it?
(but if you're gonna check the app itself in the runtime could be useful of course)

Reply:
on that note, we should probably make a simple check that the duplicated dom is the same?

Reply:
That's what the effort here https://github.com/mui/mui-toolpad/pull/1007 is about, removed that from this for now

Reply:
just saw the test changes, looks good!